spacetemplate/assets/legacy.yaml

```diff
@@ -71,7 +84,9 @@ work_item_types:
   extends: "86af5178-9b41-469b-9096-57e5155c3f31"
   name: Fundamental
   can_construct: yes
-  description: TBD
+  description: A fundamental describes the basic essentials of a software
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.

Reply:
Fixed